### PR TITLE
Add search and autocomplete for chronicles. Fix small search omnibox …

### PR DESCRIPTION
**Pull request by bbfsdev**
…issues.

### CODE REVIEW

**Comment by bbfsdev** [comment]
Demo: http://bbdev6.kbb1.com:5501/

File: src/components/Filters/FiltersHydrator.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const FiltersHydrator = ({ namespace, onHydrated }) => {
     if (isHydrated) {
       dispatch(actions.filtersHydrated(namespace));
 
-      if (onHydrated) {
+      if (!hydrated && onHydrated) {
```
Comment (gshilin) [inline src/components/Filters/FiltersHydrator.js:17]: What's `isHydrated`? Doesn't it mean that `hydrated` is true?

**Review by gshilin — CHANGES_REQUESTED** [review]

File: src/components/Filters/FiltersHydrator.js (hunk 1 above)
Comment (bbfsdev) [reply src/components/Filters/FiltersHydrator.js:17]: No, one is local state, other is redux state. They do have gaps, but If I already sent an action I don't need to hydrate the second time.

I played with the demo to check things are okay.

Comment (bbfsdev) [reply src/components/Filters/FiltersHydrator.js]: Rolled back this change. Seems I had a mistake here.

**Review by gshilin — APPROVED** [review]

**Pull request merged**